lib/insights/src/lib/analytics-process/components/analytics-report-parameters.component.ts

```diff
@@ -187,11 +186,11 @@ export class AnalyticsReportParametersComponent implements OnInit, OnChanges, On
     }
 
     convertMomentDate(date: string) {
-        return moment(date, FORMAT_DATE_ACTIVITI, true).format(FORMAT_DATE_ACTIVITI) + 'T00:00:00.000Z';
+        return DateFnsUtils.formatDate(new Date(date), FORMAT_DATE_ACTIVITI) + 'T00:00:00.000Z'
     }
 
     getTodayDate() {
```

Comment:
needs return type

Reply:
method was created by you. But I have added the return type.